source/enemies/unkindraven.js

```diff
@@ -0,0 +1,56 @@
+const { EnemyTemplate } = require("../classes");
+const { ESSENCE_MATCH_STAGGER_ALLY } = require("../constants");
+const { selectRandomAlly, selectNone } = require("../shared/actionComponents");
+const { changeStagger, generateModifierResultLines, addModifier } = require("../util/combatantUtil");
+const { spawnEnemy } = require("../util/roomUtil");
+
+module.exports = new EnemyTemplate("Unkind Raven",
```

Comment:
I like the design on these guys!

Reply:
Thank you. I wanted raven to be there
and as they are they are "Progressively everywhere and get in the way, but surprisingly aren't probabilistically guaranteed to stop you, ESPECIALLY with the Sword of the Sun"